helm/kvm-operator/templates/configmap.yaml

```diff
@@ -49,13 +49,13 @@ data:
       registry:
         domain: '{{ .Values.registry.domain }}'
         {{ if .Values.registry.mirrors -}}
-        mirrors: 
+        mirrors:
         {{- range $i, $e := .Values.registry.mirrors }}
-        - {{- $e | quote -}}
-        {{ end }}
-        {{ else }}
+        - {{ $e | quote -}}
+        {{ end -}}
+        {{ else -}}
         mirrors: []
-        {{ end }}
+        {{- end }}
```

Comment:
I can't guarantee that but this should be correct:

```
        {{- if .Values.registry.mirrors -}}
        mirrors:
        {{- range $i, $e := .Values.registry.mirrors }}
        - {{ $e | quote -}}
        {{- end }}
        {{- else }}
        mirrors: []
        {{- end }}
```

But feel free to ignore if you tested your version already

Reply:
This configuration works, just installed from control plane test catalog. Thanks!